scrapers/distgit.py

```diff
@@ -42,16 +44,50 @@ def run(self, since=None, until=None):
             end_date = timestamp_to_date(until)
         results = self.get_distgit_data(start_date, end_date)
         log.info('Successfully fetched {0} results from Teiid'.format(len(results)))
-        self.update_neo4j(results)
+        # Upload the results to Neo4j
+        _update_neo4j_partial = partial(
+            self._update_neo4j, neomodel_config.DATABASE_URL, len(results))
+        pool = Pool(2)
```

Comment:
So, we have a multi-process pool of size two handing out chunks of work to two thread pools of size 8 below...

That's complicated!  It deserves an inline comment in the code for the next person who reads the code and doesn't notice that `Pool` is not the same as `ThreadPool`.